go.sum

```diff
@@ -16,6 +16,7 @@ github.com/logrusorgru/aurora v0.0.0-20191017060258-dc85c304c434/go.mod h1:7rIyQ
 github.com/magiconair/properties v1.8.0/go.mod h1:PppfXfuXeibc/6YijjN8zIbojt8czPbwD3XqdrwzmxQ=
 github.com/mattn/go-runewidth v0.0.6 h1:V2iyH+aX9C5fsYCpK60U8BYIvmhqxuOL3JZcqc1NB7k=
 github.com/mattn/go-runewidth v0.0.6/go.mod h1:H031xJmbD/WCDINGzjvQ9THkh0rPKHF+m2gUSrubnMI=
+github.com/mesosphere/bun v1.10.1 h1:rL/orN/zmCjUJX38KjdI+qr+IMhVefV6bFjX91DBxvU=
```

Comment:
Thanks for the new check, @bbannier! Please, remove this dependency, and we'll merge the PR.